core/build.gradle

```diff
@@ -161,18 +160,17 @@ dependencies {
             "org.geotools:gt-wms:30.2",
             "org.geotools.xsd:gt-xsd-gml3:30.2",
             "org.geotools:gt-svg:30.2",
-            "org.geotools:gt-cql:30.2"
+            "org.geotools:gt-cql:30.2",
     )
     jasper(
             "net.sf.jasperreports:jasperreports:6.21.3",
-            "org.codehaus.groovy:groovy-all:3.0.22",
-            'ar.com.fdvs:DynamicJasper:5.3.9',
             "net.sf.jasperreports:jasperreports-fonts:6.21.3",
             "net.sf.jasperreports:jasperreports-functions:6.21.3",
-            'joda-time:joda-time:2.12.7'
+            "org.codehaus.groovy:groovy-all:3.0.22",
```

Comment:
Since we are not doing variable interpolation inside our dependencies, and since you are changing a lot of the dependencies: It would make sense to make them consistent. So I would recommend to use ' instead of ".

Reply:
> So I would recommend to use ' instead of ".

What's do you mean with that?

Reply:
"net.sf.jasperreports:jasperreports-functions:6.21.3",
'joda-time:joda-time:2.12.7'
"org.codehaus.groovy:groovy-all:3.0.22"
is inconsistent. I recopmmend we use 
'net.sf.jasperreports:jasperreports-functions:6.21.3'
'joda-time:joda-time:2.12.7'
'org.codehaus.groovy:groovy-all:3.0.22'
since we don't use code like that:
"net.sf.jasperreports:jasperreports-functions:$jasperreport_version_nbr",

Reply:
What's the différance? Just removing the ending comma?

For the variable I remove them because it didn't work well with Renovate!